libvast/src/system/index.cpp

```diff
@@ -869,8 +869,15 @@ index(index_actor::stateful_pointer<index_state> self,
       }
       self->state.inmem_partitions.drop(partition_id);
       self->state.persisted_partitions.erase(partition_id);
-      self->request(self->state.meta_index, caf::infinite, atom::erase_v,
-                    partition_id);
+      self
+        ->request(self->state.meta_index, caf::infinite, atom::erase_v,
+                  partition_id)
+        .then([](atom::ok) { /* nop */ },
+              [partition_id](const caf::error& err) {
+                VAST_WARN("index encountered an error trying to erase "
+                          "partition {} from the meta index: '{}",
```

Comment:
There's a `'` before the error that shouldn't be there. Also, should index not be inferred from the `self` handle?

Reply:
Technically yes, but I didn't want to open a potential can of worms where the meta index is blocked for whatever reason and a stray resume-handler is keeping a reference to the index actor. And since we wanted to move away from `self` as first argument anyways I thought this might be a good place to start.

Reply:
Alrighty.